tests/installation/partitioning/suggested_partitioning/verify_decrypted_partition_deleted.pm

```diff
@@ -0,0 +1,43 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
```

Comment:
no need for this line with the year afik for new tests, check some other new recent tests.